cico_build_deploy.sh

```diff
@@ -16,7 +16,7 @@ function setup_build_environment() {
     [ -f inherit-env ] && . inherit-env
 
     # We need to disable selinux for now, XXX
-    /usr/sbin/setenforce 0
+    /usr/sbin/setenforce 0 || :
```

Comment:
> some general questions, i bet you tested it and all works wonderfully

Well, it's hard to test, so we may be missing some things, jenkins will help us here :)

> just for my own info we are able to don't disable selinux anymore?

Basically the thing is that those repos that are RHEL enabled will be built twice. This is the first build, done as usual, using whatever is in the repo, generally CentOS based:
https://github.com/openshiftio/openshiftio-cico-jobs/blob/60200e74bf180db4eb852dc2e26c1cc4ed2ae2a4/devtools-ci-index.yaml#L546

So far no changes. However now now we have added a second build if the TARGET=rhel:
https://github.com/openshiftio/openshiftio-cico-jobs/blob/60200e74bf180db4eb852dc2e26c1cc4ed2ae2a4/devtools-ci-index.yaml#L557

But as you can see, this targets `ssh_cmd="ssh $sshopts $CICO_hostname"`, so they are being built *twice in the same node*. This means that the `cico_build_deploy.sh` script or whatever will be run twice as opposed to a single time as it used to. And setenforce will fail if you run it twice:

```shell
[root@build-rhel ~]# getenforce 
Disabled
[root@build-rhel ~]# setenforce 0
setenforce: SELinux is disabled
[root@build-rhel ~]# echo $?
1
```

Which is fixed by:

```shell
[root@build-rhel ~]# setenforce 0 || :
setenforce: SELinux is disabled
[root@build-rhel ~]# echo $?
0
```

So it doesn't fail when run twice, since we have `set -e` [here](https://github.com/jmelis/fabric8-jenkins-idler/blob/8860c9621df3f426932534f877f1363570b897cb/cico_build_deploy.sh#L5).